testnets/sgetestnet4/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "sgetestnet4",
+  "assets": [
+    {
+      "description": "The SGE token is primarily a governance token for the SGE Network blockchain.",
+      "denom_units": [
+        {
+          "denom": "usge",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "usge",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "usge",
+      "name": "SGE Testnet4",
```

Comment:
SGE testnet 4 is fine, although, would if this is not going to be the primary Testnet, then I'd add an 'additional-mintage' trace to the primary SGETestnet.
However is this is going to be primary and the ibc connections retained, I'd say this should renamed to 'sgetestnet', while the current sgetestnet should be renamed to 'sgetestnet3'.
Either is fine, but I would like to add traces here.